### PR TITLE
[price_pusher] Add near command

### PR DESCRIPTION
**Pull request by MagicGordon**

### CODE REVIEW

File: price_pusher/src/near/near.ts [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+      args: {
+        data
+      },
+      gas: "300000000000000" as any,
+      attachedDeposit: updateFee
+    });
+  }
+
+  private getConnection(network: string, accountId: string, nodeUrl: string): Connection {
+    const content = fs.readFileSync(path.join(os.homedir(), ".near-credentials", network, accountId + ".json"));
```
Comment (ali-bahjati) [inline price_pusher/src/near/near.ts:174]: This is not very good. Could you get the path from the arguments like other pushers and use that?

**Review by ali-bahjati — CHANGES_REQUESTED** [review]
Thanks a lot for contributing to us. This code looks great. I will approve after my inline comment is addressed.

File: price_pusher/src/near/near.ts (hunk 1 above)
Comment (MagicGordon) [reply price_pusher/src/near/near.ts]: Okay, added the optional parameter 'private-key-path'.

Comment (MagicGordon) [reply price_pusher/src/near/near.ts]: This path is where NEAR by default stores private keys, and users are highly likely to place their private keys here. Additionally, the official near-cli also reads private keys from this location. Therefore, the parameter for specifying the private key file path has been set as optional.

**Review by ali-bahjati — APPROVED** [review]

**Pull request merged**